vfork() crashes on some i686-compatible Synology NASes,
so use fork(), which appears to work fine, instead
--- a/libopkg/xsystem.c
+++ b/libopkg/xsystem.c
@@ -38,7 +38,7 @@ xsystem(const char *argv[])
 	int status;
 	pid_t pid;
 
-	pid = vfork();
+	pid = fork();
 
 	switch (pid) {
 	case -1:
